inst/shiny/DiagnosticsExplorer/tests/test-OrphanConceptsModule.R

```diff
@@ -0,0 +1,57 @@
+test_that("Orphan Concepts Page", {
+  initializeEnvironment(shinySettings,
+                        table1SpecPath = table1SpecPath,
+                        dataModelSpecificationsPath = dataModelSpecificationsPath)
+  
+  # Environment should have initialized
+  expect_true(exists("dataSource"))
+  
+  shiny::testServer(orphanConceptsModule, args = list(
+    id = "testOrphanConcepts", #Any string is ok?
+    dataSource = dataSource,
+    selectedCohorts = shiny::reactive("Any String"),
+    selectedDatabaseIds = shiny::reactive("Eunomia"),
+    targetCohortId = shiny::reactive({c(14906)}),
+    selectedConceptSets = NULL,
```

Comment:
Looks like this is why the test above fails, selectedConceptSets will likely need to conform to a specific model